pages/detailed-example-workflow/Step_5_apply_to_db.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: Apply to Database
+parent: Detailed Example Workflow
+nav_order: 4
```

Comment:
Should this really be 4? I don't think so.

Reply:
You're right. Should be 5